### DISCUSSION TITLE
RequestError: connect ECONNRESET with function Map

### DISCUSSION THREAD

**Discussion by Fsayagosouthern** [webdriverio/webdriverio#7588] [Q&A]
Hi everyone,

I'm using the function map(), but I always get the same error: "RequestError: connect ECONNRESET". I'm working with a calendar of React

I share with you the selector:

get day() {
    return $$('.day');
  }

The method:

async selectDate() {
    const d = new Date();
    const actualDay = d.getDate();
    var days = await this.day;

    return Promise.all(
      days.map(async (elem) => {
        const text = await elem.getText();
        return text;
      })
    );

If you need more information, only tell me. 

Thank you guys!

**Comment by christian-bromann** [comment]
How many elements `days` has? I believe if you make to many parallel requests the driver responds with `ECONNRESET`.

**Reply by Fsayagosouthern** [reply]
"Days" has 31 elements, but my idea was, get all those elements and print them in console, after that I should use those numbers.

**Reply by christian-bromann** [reply]
Can you try to print these elements is a for loop and therefor request the values sequentially? Again, my hunch here is that Chromedriver can't handle 32 parallel requests.
